### DISCUSSION TITLE
zh/guide/api/auth

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#287] [Announcements]
# zh/guide/api/auth

POST token获取 POST /api/auth/login 获取某个用户的临时JWt token Body 请求参数 请求参数 名称位置类型必选中文名说明 ------------------ bodybodyobject 否 |none » usernamebodystring 是 用户名用户名 » passwordbodystring 是 ...

https://alist.nn.ci/zh/guide/api/auth.html

**Comment by qq80284445** [comment]
大佬，如何自己加个退出按钮，点击退出呢？每次退出都要点到管理页面，再退出，好麻烦

**Comment by xianger2017** [comment]
临时的有效期是多久？

**Reply by Kuingsmile** [reply]
默认是48小时，我会补充到文档说明中

**Comment by czt6769** [comment]
是不是可以每次获取数据都可以登录，没有过48小时就会拿旧的token，这个冲不冲突

**Comment by moyanj** [comment]
请问用户信息中的role都有什么取值，有什么含义
